core/domain/stats_jobs_continuous.py

```diff
@@ -75,8 +75,8 @@ def map(item):
         Yields:
             dict(str, str). The submitted answer in dict format.
         """
-        if InteractionAnswerSummariesMRJobManager.entity_created_before_job_queued( # pylint: disable=line-too-long
-                item):
+        if (InteractionAnswerSummariesMRJobManager
```

Comment:
Don't we usually break after the first ( ?

Reply:
Yup, but I find this easier to read + added benefit of fitting in 80 chars